xrpl4j-model/src/test/java/org/xrpl/xrpl4j/model/ledger/LedgerHeaderJsonTests.java

```diff
@@ -0,0 +1,82 @@
+package org.xrpl.xrpl4j.model.ledger;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.google.common.primitives.UnsignedLong;
+import org.junit.Test;
+import org.xrpl.xrpl4j.model.AbstractJsonTest;
+import org.xrpl.xrpl4j.model.client.common.LedgerIndex;
+import org.xrpl.xrpl4j.model.transactions.Hash256;
+import org.xrpl.xrpl4j.model.transactions.TransactionType;
+
+public class LedgerHeaderJsonTests extends AbstractJsonTest {
+
+  @Test
+  public void deserializeLedgerHeaderWithTransactions() throws JsonProcessingException {
```

Comment:
Can we add test coverage around serialize -> deserialize? Or is that perhaps covered elsewhere?